.github/workflows/reporting-infrastructure.yml

```
@@ -0,0 +1,37 @@
+name: Rollout reporting and dashboarding infrastructure
+
+on:
+  push:
+    branches:
+      - main
+      - feature/TD-553-security-reprts-dashboards
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  infrastructure:
+    name: Deploy infrastructure
+    defaults:
+      run:
+        working-directory: terraform/
+    runs-on: ubuntu-latest
+    environment: prd
+    concurrency: prd
+    steps:
+      - uses: actions/checkout@v3
+      - uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_wrapper: false
+      - uses: autero1/action-terragrunt@v1.2.0
```

Als hier toch al zit te editen kan je gelijk deze updaten:
```suggestion
      - uses: autero1/action-terragrunt@v1.3.0
```
Scheelt weer een dependabot PR